src/machine/machined-dbus.c

```diff
@@ -863,7 +863,7 @@ static int method_set_pool_limit(sd_bus_message *message, void *userdata, sd_bus
                 return 1; /* Will call us back */
 
         /* Set up the machine directory if necessary */
-        r = setup_machine_directory(error);
+        r = setup_machine_directory(error, true, true);
```

Comment:
for boolean arguments to functions that you specify literally as true/false it is really hard to grok without looking at the function definition what this actually is for. We thus nowadays try to prefix such opaque arguments when calling with `/* foobar= */` indicating what this parameter is for. i.e, plese change to:

```c
r = setup_machine_directory(error, /* setup_subvol= */ true, /* setup_quota= */ true);
``

Reply:
Indeed it is. I guess I wasn't following the coding style changes closely enough.